Report for assignment1.py start evaluation at <built-in method now of type object at 0x7f1b4dd48900>
Errors occurred during run at <built-in method now of type object at 0x7f1b4dd48900>!
b'Traceback (most recent call last):\n  File "assignment1.py", line 2, in <module>\n    from msilib import type_valid\nModuleNotFoundError: No module named \'msilib\'\n'
*** ASSIGNMENT assignment1.py STATUS: FAILED ***
